helpers/fstab.py

```diff
@@ -154,4 +149,5 @@ def add(self, device, mountpoint, filesystem, options=None, dump=None, pass_=Non
         :param pass_: order to check filesystem at reboot time
         :return:
         """
-        return self.add_entry(FstabHelper.Entry(device, mountpoint, filesystem, options, dump))
+        with open(cls._path, 'a+') as fh:
```

Comment:
This does not check whether the fstab entry is already present. Might lead to duplicates and could result in fstab freaking out